openstack/networking/v2/extensions/layer3/routers/results.go

```diff
@@ -8,7 +8,15 @@ import (
 // GatewayInfo represents the information of an external gateway for any
 // particular network router.
 type GatewayInfo struct {
-	NetworkID string `json:"network_id"`
+	NetworkID        string            `json:"network_id"`
+	ExternalFixedIPs []ExternalFixedIP `json:"external_fixed_ips,omitempty"`
+}
+
+// ExternalFixedIP is the IP address and subnet ID of the external gateway of a
+// router.
+type ExternalFixedIP struct {
+	IP       string `json:"ip_address"`
```

Comment:
Let's change this to `IPAddress` for [consistency](https://github.com/gophercloud/gophercloud/blob/master/openstack/networking/v2/ports/results.go#L45-L49)

Reply:
@jtopjian Thanks a lot for digging through Neutron and finding that reference!

I've renamed `IP` to `IPAddress` as requested.